assets/app/scripts/controllers/project.js

```diff
@@ -12,6 +12,7 @@ angular.module('openshiftConsole')
     $scope.projectName = $routeParams.project;
     $scope.project = {};
     $scope.projectPromise = $.Deferred();
```

Comment:
are we using the promises on the controller any more? if not, remove both of them